internal/server/openvpn.go

```diff
@@ -141,3 +143,30 @@ func (h *openvpnHandler) getPortForwarded(w http.ResponseWriter) {
 		w.WriteHeader(http.StatusInternalServerError)
 	}
 }
+
+func (h *openvpnHandler) setPortForwarded(w http.ResponseWriter, r *http.Request) {
+	var data portsWrapper
+
+	decoder := json.NewDecoder(r.Body)
+	if err := decoder.Decode(&data); err != nil {
+		http.Error(w, err.Error(), http.StatusBadRequest)
+		return
+	}
+
+	if len(data.Ports) == 0 {
+		http.Error(w, "no port specified", http.StatusBadRequest)
+		return
+	}
```

Comment:
Maybe we could handle that as "remove forwarded ports"?

Reply:
Simply removing the statement with 3b633e435539705c25b4c1e24e862dc78d550564 will clear forwarded ports, because we already remove the old port forwards when setting the new ones anyways.